es6/003_arrow_function/arrowFunction.js

```diff
@@ -0,0 +1,11 @@
+// Mendeklarasikan dan menginisialisasi array ke dalam variabel
+const islands = [
+    'Sumatera',
+    'Jawa',
+    'Kalimantan',
+    'Sulawesi',
+    'Papua'
+];
+
+// Menampilkan panjang karakter dari setiap isi array
+console.log((islands.map(island => island.length))); // [8, 4, 10, 8 , 5]
```

Comment:
Ada kelebihan tanda kurung sama ada yang kurang

```suggestion
console.log(islands.map((island) => island.length)); // [8, 4, 10, 8 , 5]
```

Reply:
siap ane fix kan dlu

Reply:
sip :+1:

Reply:
Kalo udah `Resolve conservation` nya ya